lib/qa/authorities/linked_data/config/config_merge.rb

```diff
@@ -0,0 +1,76 @@
+module Qa::Authorities
+  module LinkedData
+    module ConfigMerge
```

Comment:
It will take a bit for me to work through all your comments.  To keep the conversation going, here is the yardoc for the module...

```
# The methods of this modules merge a partial linked data configuration into a QA provided full configuration.  QA full configurations
# are defined in QA/config/authorities/linked_data.  Partial configurations are defined in apps at APP/config/authorities/linked_data.
# Configurations are read in during application initialization by QA/config/initializers/linked_data_autorities.rb.  If a partial is
# defined by an app, the initializer will call #merge.
```